src/pages/events-description/index.js

```diff
@@ -24,25 +25,66 @@ export const EventsPageDescription = () => {
   const [event, setEvent] = useState([])
   const [thumbsSwiper, setThumbsSwiper] = useState(null)
   const [galeria, setGaleria] = useState([])
+  const [activities, setActivities] = useState([])
   const { id } = useParams()
+  const activitiesList = []
   useEffect(() => {
-    cms.get(`api/events/${id}/?populate=parceires&populate=fotos_evento`).then((response) => {
-      const { data } = response.data
-      setEvent(data)
+    cms
+      .get(
+        `api/events/${id}/?populate=parceires
+        &populate=fotos_evento
+        &populate=subeventos
+        &populate=oficinas
+        &populate=oficinas.foto_divulgacao
+        &populate=subeventos.foto_divulgacao`
+      )
```

Comment:
O Axios suporta [configuração de query parameters](https://github.com/axios/axios#request-config) como um objeto, assim não precisamos concatenar strings. Fica mais flexível e fácil de manter:

```suggestion
cms.get(`api/events/${id}/`, {
  params: {
    populate: [
      "parceires",
      "fotos_evento",
      "subeventos",
      "oficinas",
      "oficinas.foto_divulgacao",
      "subeventos.foto_divulgacao",
    ],
  },
});
```

Fica um pouco mais fácil de ler também, pois não precisamos ficar repetindo `&populate` toda vez.